.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea for GPflow
+labels: enhancement
+---
+
+<!-- Lines like this are comments and will be invisible -->
+
+# Feature request
+
+<!-- Please give a clear and concise description of your feature proposal! -->
+
+## Motivation
+
+**Is your feature request related to a problem?**
+<!-- A clear and concise description of the problem. For example: I'm always frustrated when [...] -->
+<!-- Are there relevant issues or other PRs? Please add links! -->
+<!-- Who will benefit from this feature? -->
+
+## Proposal
+
+**Describe the solution you would like**
+<!-- A clear and concise description of what you want to happen. -->
+
+**What alternatives have you considered?**
+<!-- A clear and concise description of any alternative solutions or features you've considered. -->
+
+**Are you willing to open a pull request?** (We really appreciate contributions!)
```

Comment:
I'm not sure about the later part - it sounds unprofessional

Reply:
how could it sound more professional while still showing to users that we would really like them to turn into contributors rather than expecting volunteers to do all the work?

Reply:
this is fine for now. perhaps i'll come back to the whole approach of getting contributors